updatehub/src/states/prepare_local_install.rs

```diff
@@ -28,7 +28,8 @@ impl StateChangeImpl for State<PrepareLocalInstall> {
         info!("Prepare local install: {:?}", self.0.update_file);
         let dest_path = shared_state.settings.update.download_dir.clone();
         std::fs::create_dir_all(&dest_path)?;
-        compress_tools::uncompress(self.0.update_file, &dest_path, compress_tools::Kind::Zip)?;
+        compress_tools::uncompress(self.0.update_file, &dest_path, compress_tools::Kind::Zip)
+            .map_err(|e| super::TransitionError::Uncompress(e))?;
```

Comment:
Only `.map_err(super::TransitionError::Uncompress)` should suffice here.

Reply:
Done.